docs/content/en/docs/implementing/evaluatemetrics.md

```diff
@@ -1,5 +1,144 @@
+
+Use the Kubernetes Custom Metrics API
+to refer to `KeptnMetric` via the
+[Kubernetes HorizontalPodAutoscaler](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)
+(HPA),
+as in the following example:
+
+```yaml
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: podtato-head-entry
+  namespace: podtato-kubectl
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: podtato-head-entry
+  minReplicas: 1
+  maxReplicas: 10
+  metrics:
+    - type: Object
+      object:
+        metric:
+          name: keptnmetric-sample
+        describedObject:
+          apiVersion: metrics.keptn.sh/v1alpha1
+          kind: KeptnMetric
+          name: keptnmetric-sample
+        target:
+          type: Value
+          value: "10"
+```
+
+See the [Scaling Kubernetes Workloads based on Dynatrace Metrics](https://www.linkedin.com/pulse/scaling-kubernetes-workloads-based-dynatrace-metrics-keptnproject/)
+blog post
+for a detailed discussion of doing this with Dynatrace metrics.
+The same approach could be used to implement HPA with other data providers.
+
+### Retrieve KeptnMetric values with kubectl
+
+Use the `kubectl get --raw` command
+to retrieve the values of a `KeptnMetric`, as in the following example:
+
+```shell
+$ kubectl get --raw "/apis/custom.metrics.k8s.io/v1beta2/namespaces/podtato-kubectl/keptnmetrics.metrics.sh/keptnmetric-sample/keptnmetric-sample" | jq .
+
+{
+  "kind": "MetricValueList",
+  "apiVersion": "custom.metrics.k8s.io/v1beta2",
+  "metadata": {},
+  "items": [
+    {
+      "describedObject": {
+        "kind": "KeptnMetric",
+        "namespace": "podtato-kubectl",
+        "name": "keptnmetric-sample",
+        "apiVersion": "metrics.keptn.sh/v1alpha1"
+      },
+      "metric": {
+        "name": "keptnmetric-sample",
+        "selector": {
+          "matchLabels": {
+            "app": "frontend"
+          }
+        }
+      },
+      "timestamp": "2023-01-25T09:26:15Z",
+      "value": "10"
+    }
+  ]
+}
+```
+
+### Filter on matching labels
+
+You can filter based on matching labels.
+For example, to retrieve all metrics
+that are labelled with `app=frontend`,
+use the following command:
+
+```shell
+$ kubectl get --raw "/apis/custom.metrics.k8s.io/v1beta2/namespaces/podtato-kubectl/keptnmetrics.metrics.sh/*/*?labelSelector=app%3Dfrontend" | jq .
+
+{
+  "kind": "MetricValueList",
+  "apiVersion": "custom.metrics.k8s.io/v1beta2",
+  "metadata": {},
+  "items": [
+    {
+      "describedObject": {
+        "kind": "KeptnMetric",
+        "namespace": "keptn-lifecycle-toolkit-system",
+        "name": "keptnmetric-sample",
+        "apiVersion": "metrics.keptn.sh/v1alpha3"
+      },
+      "metric": {
+        "name": "keptnmetric-sample",
+        "selector": {
+          "matchLabels": {
+            "app": "frontend"
+          }
+        }
+      },
+      "timestamp": "2023-01-25T09:26:15Z",
+      "value": "10"
+    }
+  ]
+}
+```
```

Comment:
I would move this part under "Accessing Metrics via the Kubernetes Custom Metrics API" since it shows how to access the metric through the Kubernetes APIs

Reply:
It actually IS under that heading -- it's just the third subsection  -- look in the upper left of the netlify view.  I'm thinking that maybe the HPA subsection should be a level ## after this.  Yeah, it's technically part of "... via the Kubernetes Custom Metrics API" but...  If I move HPA, then this section has the "retrieve ... with kubectl" followed by "Filter on..."  That looks more logical

Reply:
oh, you are right! My bad I misread the heading 👍